Corona-Warn-App/src/main/res/layout/fragment_settings_tracing.xml

```diff
@@ -16,14 +16,18 @@
         <variable
             name="settingsViewModel"
             type="de.rki.coronawarnapp.ui.viewmodel.SettingsViewModel" />
+
+        <variable
+            name="interopViewModel"
```

Comment:
`interoperabilityViewModel`

Reply:
variable was redundant anyway :D Removed it completly